packages/plugin-axe/project.json

```diff
@@ -6,7 +6,17 @@
   "tags": ["scope:plugin", "type:feature", "publishable"],
   "// targets": "to see all targets run: nx show project plugin-axe --web",
   "targets": {
-    "build": {},
+    "install-browser": {
+      "command": "playwright-core install chromium",
+      "cache": true,
+      "inputs": [
+        "sharedGlobals",
+        { "runtime": "playwright-core install chromium --dry-run" }
+      ]
+    },
+    "build": {
+      "dependsOn": ["^build", "install-browser"]
+    },
```

Comment:
I like that the browser installation is now part of our Nx graph. 👍

However, I'm wondering if `build` is the right target where we should define the dependency. We don't need have the browser installed in order to build the package. So in some scenarios (e.g., checking for type errors), we may be adding an unnecessary and expensive operation. 

As I understand, the browser installation is only required for our `code-pushup-axe` target (you've already defined this dependency in our root `project.json`) and for `e2e-test` target of `plugin-axe-e2e`. So I'm thinking it would be more optimal to add `^install-browser` to the `dependsOn` array of the `e2e-test` target instead. This could even be applied globally for all `e2e-test` targets 🤔

Reply:
I see. Cool, that makes things simpler :+1: